src/Command/InitCommand.php

```diff
@@ -22,16 +22,10 @@
 
 class InitCommand extends Command
 {
-    private UserPasswordEncoderInterface $passwordEncoder;
-    private ManagerRegistry $registry;
-
     protected static $defaultName = 'starfleet:init';
 
-    public function __construct(UserPasswordEncoderInterface $passwordEncoder, ManagerRegistry $registry)
+    public function __construct(private UserPasswordEncoderInterface $passwordEncoder, private ManagerRegistry $registry)
```

Comment:
I don't really like this shape because it scales badly, and now it's harder to find all class properties.

Could you edit all constructor to use the following shape:

```suggestion
    public function __construct(
        private UserPasswordEncoderInterface $passwordEncoder,
        private ManagerRegistry $registry,
    ) {
```

Don't miss the last trailing coma 

Even for constructor with only one arguments: This will avoid to mess up the diff when adding a new argument